docs/source/examples/classification_and_regression.rst

```diff
@@ -0,0 +1,302 @@
+    dataset_folder = f'{data_root}/celeba'
+    os.makedirs(dataset_folder, exist_ok=True)
+
+    for file in file_list:
+        url = f"{base_url}/{file}"
+        if not os.path.exists(f"{dataset_folder}/{file}"):
+        wget.download(url, f"{dataset_folder}/{file}")
+
+    with zipfile.ZipFile(f"{dataset_folder}/img_align_celeba.zip", 'r') as ziphandler:
+        ziphandler.extractall(dataset_folder)
+   
+Now, as the dataset id downloaded, we can define our datasets and dataloaders in its original way.
+
+.. code-block:: python
+
+    transforms = tfms.Compose ([
+                            tfms.Resize((image_size, image_size)),
+                            tfms.ToTensor(),
+                            tfms.Normalize(imagenet_mean, imagenet_std)
+    ])
+    train_dataset = datasets.CelebA(data_root, 
+                                    split='train', 
+                                    target_type=['attr', 'landmarks'], 
+                                    transform=transforms)
+    valid_dataset = datasets.CelebA(data_root, 
+                                    split='valid', 
+                                    target_type=['attr', 'landmarks'], 
+                                    transform=transforms)
+    test_dataset = datasets.CelebA(data_root, 
+                                split='test', 
+                                target_type=['attr', 'landmarks'], 
+                                transform=transforms)
+                                
+    train_dataloader = DataLoader(train_dataset, batch_size=batch_size, shuffle=True)
+    valid_dataloader = DataLoader(valid_dataset, batch_size=batch_size, shuffle=False)
+    test_dataloader = DataLoader(test_dataset, batch_size=batch_size, shuffle=False)
+
+Here we can see how each dataset sample looks like:
+
+.. code-block:: python
+
+    print (train_dataset[0])
+
+Regarding the complexity of the problem and the high number training/valid samples, we can seperate and use a portion of data as below:
+
+.. code-block:: python
+
+    train_subset = Subset(train_dataset, np.arange(1,10000))
+    valid_subset = Subset(valid_dataset, np.arange(1,2000))
+    train_dataloader = DataLoader(train_subset, batch_size=batch_size, shuffle=True)
+    valid_dataloader = DataLoader(valid_subset, batch_size=batch_size, shuffle=False)
+
+Here, we can see an example from the training dataset. It shows an image of a person, printing the gender and also showing the location of the eyes. It is worth mentioning that as we resize the image, the coordinates of the eyes should also be changed with same ratio.
+
+.. code-block:: python
+
+    sample_number = 189
+    image = train_dataset[sample_number][0]
+    image = image.permute(1,2,0).detach().numpy()
+    image_rgb = cv2.cvtColor(np.float32(image), cv2.COLOR_BGR2RGB)
+    image_rgb = image_rgb * imagenet_std + imagenet_mean
+    Gender = 'male' if int(train_dataset[sample_number][1][0][20])==1 else 'female'
```

Comment:
I'm guessing that 20 is the index of the gender. I think it should be defined as a constant and explained after its definition. Otherwise, one might be confused by the code.